components/bsx/Gallery/Item/GalleryItem.vue

```diff
@@ -0,0 +1,349 @@
+      query: query.default,
+      client: this.urlPrefix,
+      variables: {
+        id: createTokenId(this.collectionId, this.id),
+      },
+    })
+    const {
+      data: { nftEntity },
+    } = nft
+
+    if (!nftEntity) {
+      this.$consola.warn(`No NFT with ID ${this.id} fallback to RPC Node`)
+      this.fetchRPCMetadata()
+      // showNotification(`No NFT with ID ${this.id}`, notificationTypes.warn)
+      return
+    }
+
+    this.$consola.log('nft', nftEntity)
+
+    this.nft = {
+      ...this.nft,
+      ...nftEntity,
+      metadata: nftEntity.metadata,
+    }
+
+    if (nftEntity.meta) {
+      this.meta = {
+        ...this.meta,
+        ...nftEntity.meta,
+        image: sanitizeIpfsUrl(nftEntity.meta.image || ''),
+      }
+    }
+  }
+
+  protected fetchRPCMetadata() {
+    onApiConnect(async (api) => {
+      const metacall = getMetadata(api)
+      const res = await metacall(this.collectionId, this.id).then((option) =>
+        unwrapOrNull(option as Option<any>)
+      )
+      if (res) {
+        Vue.set(this.nft, 'metadata', res.metadata.toHuman())
+        this.fetchMetadata()
+      }
+    })
+  }
+
+  onImageError(e: any) {
+    this.$consola.warn('Image error', e)
+  }
+
+  public async fetchAnimationData() {
+    if (this.meta.animation_url && !this.mimeType) {
+      const { headers } = await axios.head(this.meta.animation_url)
+      this.mimeType = headers['content-type']
+      this.$consola.log(this.mimeType)
+      const mediaType = resolveMedia(this.mimeType)
+      this.imageVisible = ![
+        MediaType.VIDEO,
+        MediaType.MODEL,
+        MediaType.IFRAME,
+        MediaType.OBJECT,
+      ].some((t) => t === mediaType)
+    }
+  }
+
+  public async fetchMetadata() {
+    // this.$consola.log(this.nft);
+
+    if (this.nft['metadata'] && !this.meta['image']) {
+      const cachedMeta = await get(this.nft.metadata)
+
+      const meta = cachedMeta
+        ? cachedMeta
+        : await fetchNFTMetadata(
+            this.nft,
+            getSanitizer(this.nft.metadata, undefined, 'permafrost')
+          )
+      this.$consola.log(meta)
+
+      const imageSanitizer = getSanitizer(meta.image)
+      this.meta = {
+        ...meta,
+        image: imageSanitizer(meta.image),
+        animation_url: sanitizeIpfsUrl(
+          meta.animation_url || meta.image,
+          'pinata'
+        ),
+      }
+
+      if (!this.nft.name && meta.name) {
+        Vue.set(this.nft, 'name', meta.name)
+      }
+
+      this.$consola.log(this.meta)
+      if (this.meta.animation_url && !this.mimeType) {
+        const { headers } = await axios.head(this.meta.animation_url)
+        this.mimeType = headers['content-type']
+        this.$consola.log(this.mimeType)
+        const mediaType = resolveMedia(this.mimeType)
+        this.imageVisible = ![
+          MediaType.VIDEO,
+          MediaType.MODEL,
+          MediaType.IFRAME,
+          MediaType.OBJECT,
+        ].some((t) => t === mediaType)
```

Comment:
`![].some(t => t === mediaType)` -> `[].every(t => t !== mediaType)`
Not (Some of t equal to mediaType) -> every t not equal to mediaType

maybe the latter one would be more forth straight?